_topics/en/bech32.md

```diff
@@ -16,8 +16,8 @@ excerpt: >
 
 ## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
 extended_summary: |
-  Using only 32 letters and numbers, the bech32 address format is case
-  insensitive and includes an error-correction code that can catch
+  Using only 32 letters and numbers, the bech32 address format does not use
+  mixed case and includes an error-correction code that can catch
```

Comment:
maybe "is case insensitive (but mixed case is invalid)"

Reply:
I was considering phrasing it as "is single case", but I wasn't sure whether that would be as clear.

Reply:
Agree, would've scratched my head with that one.